vllm/plugins/__init__.py

```diff
@@ -14,6 +14,14 @@
 plugins_loaded = False
 
 
+def _force_torch_inductor_compile_threads(thread_num: int):
+    import torch
```

Comment:
I think `import torch` at the top of this file should be fine.

Simply add `torch._inductor.config.compile_threads = 1` inside `load_general_plugins`

Reply:
Done